modules/end-user-guide/pages/adding-a-vscode-extension.adoc

```diff
@@ -1,162 +1,50 @@
-
-If you use the Che-Theia IDE, the extension is installed and configured automatically. If you use a different supported IDE with the Che-Code Visual Studio Code fork, the IDE displays a pop-up with a recommendation to install the extension. 
-
-.Prerequisites
-. You have the `.vscode/extensions.json` file in the root of the GitHub repository.
-
-.Procedure
-. Add the extension ID to the `.vscode/extensions.json` file. Use a `.` sign to separate the publisher and extension. The following example uses the IDs of Red Hat Visual Studio Code Java extension:
-+
-[source,json,subs="+quotes"]
-----
-  {
-    "recommendations": [
-      "redhat.java"
-    ]
-  }
-----
-
-NOTE: If the specified set of extension IDs isn't available in the {prod-short} registry, the workspace starts without the extension.
-
-
-[id="che-theia-plugins-YAML"]
-== Adding plugin parameters to `.che/che-theia-plugins.yaml`
-You can add extra parameters to a plugin by modifying the `.che/che-theia-plugins.yaml` file. These modifications include:
-
-* Defining the plugins for workspace installation.
-* Changing the default memory limit.
-* Overriding default preferences.
-
-
-=== Defining the plugins for workspace installation
-Define the plugins to be installed in the workspace.
-
-.Prerequisites
-. You have the `.che/che-theia-plugins.yaml` file in the root of the GitHub repository.
-
-.Procedure
-. Add the ID of the plugin to the  `.che/che-theia-plugins.yaml` file. Use the `/` sign to separate the publisher and plugin name. The following example uses the IDs of Red Hat Visual Studio Code Java extension:
-+
-[source,yaml,subs="+quotes"]
-----
-- id: redhat/java/latest
-----
-
-=== Changing the default memory limit
-
-Override container settings such as the memory limit.
+{prod-short} can automatically install workspace recommended extensions.
 
 .Prerequisites
-. You have the `.che/che-theia-plugins.yaml` file in the root of the GitHub repository.
-
-.Procedure
-. Add an `override` section to the `.che/che-theia-plugins.yaml` file under the `id` of the plugin.
-. Specify the memory limit for the extension. In the following example, {prod-short} automatically installs the Red Hat Visual Studio Code Java extension in the {prod-short} workspace and increases the memory of the workspace by two gigabytes:
+* {prod-short} has access to the link:https://www.open-vsx.org/[OpenVSX registry].
 +
-[source,yaml,subs="+quotes"]
-----
-
-  - id: redhat/java/latest
-    override:  
-      sidecar:  
-        memoryLimit: 2Gi
-----
+[TIP]
+====
+To install recomended extensions in a restricted environment, consider these options:
 
-=== Overriding default preferences
+* xref:selecting-a-workspace-ide.adoc[] that accesses an internal plugin registry, such as Che-Theia.
 
-Override the default preferences of the Visual Studio Code extension for the workspace.
+* xref:administration-guide:configuring-the-open-vsx-registry-url.adoc[] to point to your OpenVSX registry.
```

Comment:
OpenVSX registry works only with Che-Code editor. Maybe it's better to use Che-Code instead of Che-Theia in the first step.

Reply:
I believe the last changes are adressing the issue.